src/hooks/useMongo.js

```diff
@@ -14,6 +16,8 @@ import {AnonymousCredential} from "mongodb-stitch-core-sdk";
  */
 export const useMongo = (input) => {
 
+    const logger = useLogger();
```

Comment:
Awesome job chaining our logger hook in here. Although this is a new instance of `useLogger`, so it won't be adding logs to the same logger instance that we are passing through the rest of the app via `context`, which it doesn't necessarily need to, depending on your intent. If you want this to keep a separate history of logs just for use in the mongoHook, you might want to expose this log list or the logger display from the mongo hook, so we can view these particular logs using `mongoHook.logs` or something similar